### ISSUE TITLE
Signed projects with satellite assemblies fail

### ISSUE THREAD

**Issue by ryanbrandenburg** [dotnet/sdk#784]
If you have a project like [this](https://github.com/ryanbrandenburg/SatelliteSigningRepro) which has a satellite assembly and is signed and you run `dotnet restore; dotnet run` you get errors like:
```
C:\Users\rybrande\AppData\Local\Microsoft\dotnet\sdk\1.0.0-rc3-004530\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(213,5): error : Errors occured when emitting satellite assembly 'C:\Users\rybrande\dd\Localization\test\LocalizationWebsite\obj\Release\net451\fr-FR\LocalizationWebsite.resources.dll'. [C:\Users\rybrande\dd\Localization\test\LocalizationWebsite\LocalizationWebsite.csproj]
C:\Users\rybrande\AppData\Local\Microsoft\dotnet\sdk\1.0.0-rc3-004530\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Sdk.targets(213,5): error : error CS7027: Error signing output with public key from file 'C:\Users\rybrande\dd\Localization\build\Key.snk' -- Assembly signing not supported. [C:\Users\rybrande\dd\Localization\test\LocalizationWebsite\LocalizationWebsite.csproj]
```

**Comment by dsplaisted** [comment]
What version of the .NET CLI are you using?  I think we've recently fixed several issues similar to this.  @nguerrera

**Comment by nguerrera** [comment]
Please try with latest CLI. Looks like dupe of fixed bug #692.
